pip/requirements.txt

```diff
@@ -20,6 +20,7 @@ pkginfo==1.9.6
 pycparser==2.21
 pygments==2.14.0
 pyparsing==3.0.9
+pywin32-ctypes==0.2.2; sys_platform == 'win32'
```

Comment:
Do we need this

Reply:
Yes, this is required by keyring on windows and is not fetched automatically.